src/Interpreter.test.ts

```diff
@@ -1614,7 +1614,7 @@ test('interpret votes', async () => {
       },
       Object {
         "option": "Jane Bland",
-        "score": 0.5717884130982368,
+        "score": 0.5714285714285714,
```

Comment:
The decoding of images is slightly different between canvas and sharp, so we get different scores.